test/python/opflow/legacy/test_weighted_pauli_operator.py

```diff
@@ -100,30 +100,30 @@ def test_multiplication(self):
         pauli_b = 'ZYIX'
         coeff_a = 0.5
         coeff_b = 0.5
-        pauli_term_a = [coeff_a, Pauli.from_label(pauli_a)]
-        pauli_term_b = [coeff_b, Pauli.from_label(pauli_b)]
+        pauli_term_a = [coeff_a, Pauli(pauli_a)]
+        pauli_term_b = [coeff_b, Pauli(pauli_b)]
         op_a = WeightedPauliOperator(paulis=[pauli_term_a])
         op_b = WeightedPauliOperator(paulis=[pauli_term_b])
         new_op = op_a * op_b
 
         self.assertEqual(1, len(new_op.paulis))
-        self.assertEqual(-0.25, new_op.paulis[0][0])
-        self.assertEqual('ZZYY', new_op.paulis[0][1].to_label())
+        self.assertEqual(0.25, new_op.paulis[0][0])
+        self.assertEqual('-ZZYY', new_op.paulis[0][1].to_label())
```

Comment:
Why did the signs have to change here?

Reply:
New Pauli can have the phase information. Since 0.25 * -ZZYY = -0.25 * ZZYY , they are the same.